deploy/kustomize/operator-webhook/webhook-certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: self-signed-issuer
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: pmem-csi-operator-selfsigned  # the name used for 'cert-manager.io.inject-ca-from' CRD annotation
+spec:
+  dnsNames:
+  # dnsNames must match be: <webhook-service-name><.namespace>.svc[.domain]
+  - $(SERVICE_NAME).$(SERVICE_NAMESPACE).svc
+  - $(SERVICE_NAME).$(SERVICE_NAMESPACE).svc.cluster.local
+  secretName: pmem-csi-operator-service-cert # name referenced in: patches/operator-webhook-patch.yaml
+  issuerRef:
+    kind: Issuer
+    name: self-signed-issuer
+---
+
```

Comment:
nit: extra separator and line at the end

Reply:
Fixed.